### ISSUE TITLE
Unable to create certificate for `local.domain.com`

### ISSUE THREAD

**Issue by amiryonatan** [davewasmer/devcert#83]
Since version 1.2.1, we're getting an error:

```
Error: "local.domain.com" is not a valid domain name.
    at /Users/amiry/dev/editor-x/bundles/editorx-santa-editor-bridge/node_modules/devcert/dist/index.js:40:23
    at Array.forEach (<anonymous>)
```

**Comment by mcalthrop-lego** [comment]
@amiryonatan The latest release uses an external npm package for determining validity of a domain name, and subdomains have been explicitly disallowed.

I'm not sure of the reason for this; see my comment here: https://github.com/davewasmer/devcert/pull/79#issuecomment-1117340943

**Comment by rowong** [comment]
The change also appears to have busted generating self signed certs for "localhost".  We are seeing the following error:

```
UnhandledPromiseRejectionWarning: Error: "localhost" is not a valid domain name.
    at /mnt/c/wsl/node_modules/devcert/dist/index.js:40:23
    at Array.forEach (<anonymous>)
```

**Comment by NewFuture** [comment]
patch a fix pr #84 for this

**Comment by Loveself7070** [comment]
84#
